source/manuals/programming-languages/python/python.html.md.erb

```diff
@@ -0,0 +1,94 @@
+---
+title: Writing Python at GDS
+expires: living
+---
+
+# <%= current_page.data.title %>
+
+
+### About this manual:
+
+This manual is designed to aid developers in writing Python code that is clear and consistent, within, and across,
+projects at GDS.
+
+The [Google Python style guide][GPSG] outlines sensible conventions for
```

Comment:
(sadly) I don't write Python day-to-day at GDS however if I'm ever unsure of how I should be formatting Python I look first at PEP8. So I'd suggest we link to that here at the top of the guide.

Some things in the Google Style Guide seem worth saying e.g. "avoid writing metaclasses" however I wonder if this sentence introducing Python at GDS shouldn't first be saying "We follow PEP8, in some cases PEP8 doesn't express a view (e.g. on the usage of language features such as metaclasses) in these cases we defer to the Google Style Guide."

Reply:
Updated this, see what you think